src/generic/status-badge/StatusBadge.jsx

```diff
@@ -9,9 +9,7 @@ function StatusBadge({ intl, status, label }) {
   return (
     <>
       {label && `${label} `}
-      {status
-        ? <Badge className="py-1" variant="success">{intl.formatMessage(messages.enabled)}</Badge>
-        : <Badge className="py-1" variant="secondary">{intl.formatMessage(messages.disabled)}</Badge>}
+      {status && <Badge className="py-1" variant="success">{intl.formatMessage(messages.enabled)}</Badge>}
```

Comment:
should rename that to "StatusEnabledBadge"?

Reply:
Yes, We should change the component name.

Reply:
Also, why do we still need "status"? do you mean we can call this component without status? and want to avoid rendering it?

Reply:
Yes. Status is working as double check to avoid the unwanted rendering.

Reply:
I don't think we need double checks here?

Reply:
Okay